LightBulb.WindowsApi/PowerBroadcast.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+using LightBulb.WindowsApi.Internal;
+
+
+namespace LightBulb.WindowsApi
+{
+    public partial class PowerBroadcast
+    {
+        public Action<byte> Handler { get; }
+        public Guid Id { get; }
+
+        readonly IntPtr _handle;
+
+        public PowerBroadcast(IntPtr handle, Action<byte> handler, Guid id)
+        {
+            Handler = handler;
+            Id = id;
+            SpongeWindow.MessageReceived += SpongeWindowOnMessageReceived;
```

Comment:
You can use `SpongeWindow.Instance` so that all objects are reusing the same window.